task/yq/0.1/test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+tkn pipelinerun delete yq-pipeline-run -f -n demo-workshop-ops
+tkn pipeline delete yq-pipeline -f -n demo-workshop-ops
+tkn task delete yq -f -n demo-workshop-ops
+
+kubectl apply -n demo-workshop-ops -f yq.yaml
+kubectl apply -n demo-workshop-ops -f tests/resources.yaml
+kubectl apply -n demo-workshop-ops -f tests/run.yaml
```

Comment:
any reason for adding this file? If the objective is for running tests then we don't need this file :smile: as the testing infra checks for the tests dir and if it exists then it runs the tests

Reply:
Deleted!